### DISCUSSION TITLE
Status of the application

### DISCUSSION THREAD

**Discussion by anthraxn8b** [photoprism/photoprism#4612] [Q&A] [answered]
Hi! Is there a "page" which shows the current state of the application - like "how many images are left for import", "how many images are left for processing", "how long will it approxymately take",... This would be great to know :-)

**Comment by graciousgrey** [comment, marked as answer]
Thank you for your feedback! 

This is not possible yet. You can find a related feature request [here](https://github.com/photoprism/photoprism/issues/639)
